Admin/ArticleAdmin.php

```diff
@@ -136,13 +136,15 @@ protected function configureFormFields(FormMapper $formMapper)
             ->with('General', array('class' => 'col-md-8'))
                 ->add('title', TextType::class, array(
                     'attr' => array('maxlength' => 255),
+                    'help' => $this->trans('help_article_title'),
                 ))
                 ->add('subtitle', TextType::class, array(
                     'required' => false,
                     'attr' => array('maxlength' => 255),
                 ))
                 ->add('abstract', TextareaType::class, array(
                     'required' => false,
+                    'help' => $this->trans('help_article_abstract'),
```

Comment:
I've got another issue with helps text that doesn't display, so I will separate this PR in 2, one for the translation, one for the help texts.